src/Canvas.tsx

```diff
@@ -313,7 +302,6 @@ const InternalCanvas: FC<CanvasProps & { ref?: Ref<CanvasRef> }> = forwardRef(({
       }}
       onMouseEnter={onMouseEnter}
       onMouseLeave={onMouseLeave}
-      {...bind()}
```

Comment:
You don't need this because u set the `domTarget` correct?

Reply:
Correct